packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -74,26 +75,31 @@ gem build %{gem_name}.gemspec
 %{?scl:EOF}
 
 %install
-mkdir -p %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d
-install -m 755 .%{gem_instdir}/config/katello.yml %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d/katello.yml
 mkdir -p %{buildroot}%{gem_dir}
 cp -pa .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
+mkdir -p %{buildroot}%{hammer_confdir}/cli.modules.d
+install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
+                %{buildroot}%{hammer_confdir}/cli.modules.d/%{plugin_name}.yml
+
 %files
 %dir %{gem_instdir}
-%{gem_instdir}/lib
+%{gem_libdir}
 %{gem_instdir}/locale
-%config(noreplace) %{_root_sysconfdir}/%{confdir}/cli.modules.d/katello.yml
 %exclude %{gem_cache}
 %{gem_spec}
+%config %{hammer_confdir}/cli.modules.d/%{plugin_name}.yml
```

Comment:
`noreplace` was dropped, not sure that's correct

Reply:
I *just* had a draft i was about to submit. I've noticed this in other places as well but I think most hammer plugins actually use `noreplace`. Shall we update the template?

Reply:
`noreplace` means it creates an `.rpmnew` instead of moving the existing one to `.rpmsave`? I can never remember. But yeah. Seems sensible to update the template then.

Reply:
https://www.cl.cam.ac.uk/~jw35/docs/rpm_config.html has a nice table. `%config` is `rpmsave`. `%config(noreplace)` is `.rpmnew`.